specification/src/chapter4.adoc

```diff
@@ -49,15 +49,10 @@ non-security services.
 | Requirement
 
 | SR_GEN_001
-| PMP/ePMP, or MTT, MUST be used to isolate M-mode from lower privilege levels.
+| PMP/ePMP, MUST be used to isolate M-mode from lower privilege levels.
```

Comment:
now that I think; doesn't  MTT provide an implicit self protection of the M-mode pages by virtue of marking them as not accessible in all supervisor domains? are we mandating PMP/ePMP as defense-in-depth?

Reply:
agree - why was this requirement modified? I think PMP may be used when MTT is not available.

Reply:
It's not been modified. We kept PMP/ePMP as an option as it provides the "reverse protection". I allows M-mode to prevent itself from executing from or accessing memory assigned to lower privilege levels (privilege escalation). MTT does not have this capability as it is not (directly) related to privilege modes (and shouldn't be).

I agree this is more "defence in depth" as Osman said. I''l change it to a "MAY".

Actually, I think this is one of those where there is "duplication" from previous chapters. And that's why we have confusion as they are not in synch... I'll see if I can replace this with a reference instead.